src/model_signing/_signing/sign_certificate.py

```diff
@@ -116,8 +126,16 @@ def __init__(
 
         self._store = crypto.X509Store()
         for certificate in certificates:
+            if self._log_fingerprints:
+                self.display_cert_fingerprint(certificate, hashes.SHA256())
             self._store.add_cert(crypto.X509.from_cryptography(certificate))
 
+    def display_cert_fingerprint(self, cert, hash_alg: hashes.Hash) -> None:
```

Comment:
This does not use `self`, it can be a global free function (outside of the class).

My bad, I think I was not clear in previous review when I said to make this private, I meant to make it `_log_cert_fingerprint` (with the leading `_` so it should not be used outside of the module -- except in tests)

Reply:
Fixed.